scripts/lib.py

```diff
@@ -6,6 +6,8 @@
 DEV_REPOS = (
     "accountsservice",
     "amd-ppt-bin",
+    "alsa-ucm-conf",
+    "alsa-utils",`
```

Comment:
Now I think it has an extra backtic :sweat_smile:

Reply:
I don't even know what to say :man_facepalming: